integration_tests/integration/unallocated-by-team.spec.ts

```diff
@@ -11,27 +15,52 @@ context('Unallocated cases by team', () => {
       name: 'Team Name',
     })
     cy.visit('/team/TM1/cases/unallocated')
+    unallocatedPage = new UnallocatedByTeamPage('Team Name')
+  })
+
+  it('User name visible in header', () => {
```

Comment:
Not sure these header and footer tests should be in here. Ideally they should be tested on every page, but if not then the home page would seem best `home.spec.ts`.

What do you think?